macros/EmbedYouTube.ejs

```diff
@@ -25,7 +25,7 @@ if (ratio == "16:9") {
 
 var video = "<div class='intrinsic-wrapper'>";
 video += "<div class='intrinsic-container " + ratio_class + "'>";
-video += "<iframe allowfullscreen src='https://www.youtube.com/embed/" + $0 + "?rel=0&html5=1'></iframe>";
+video += "<iframe title='Video.' allowfullscreen src='https://www.youtube.com/embed/" + $0 + "?rel=0&html5=1'></iframe>";
```

Comment:
Change `title='Video.'` to `title="<%= s_title %>"` and add the following after line 15:
```js
let s_title = MDN.localString({
  "en-US": "YouTube Video",
});
```